nsqd/topic.go

```diff
@@ -452,10 +452,10 @@ func (t *Topic) LoadChannelMeta() error {
 		if ch.Skipped {
 			channel.Skip()
 		}
-		//skip zan test message according to meta file
-		if ch.IsZanTestSkipepd() {
-			channel.SkipZanTest()
-		} //else nothing maybe unskip
+		//unskip zan test message according to meta file
+		if !ch.IsZanTestSkipepd() {
+			channel.UnskipZanTest()
+		}
```

Comment:
I think we need to add `else` here since we do not know the old state of the channel.